editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -0,0 +1,223 @@
+import Box, { BoxProps } from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Collapse from "@mui/material/Collapse";
+import Link from "@mui/material/Link";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListSubheader from "@mui/material/ListSubheader";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import type {
+  Constraint,
+  GISResponse,
+  Metadata,
+} from "@opensystemslab/planx-core/types";
+import groupBy from "lodash/groupBy";
+import React, { ReactNode, useState } from "react";
+import ReactHtmlParser from "react-html-parser";
+import Caret from "ui/icons/Caret";
+import ReactMarkdownOrHtml from "ui/ReactMarkdownOrHtml";
+
+const CATEGORY_COLORS: Record<string, string> = {
+  "General policy": "#99C1DE",
+  "Heritage and conservation": "#EDDCD2",
+  Ecology: "#E0EFCC",
+  Trees: "#DBE7E4",
+};
+
+interface StyledConstraintProps extends BoxProps {
+  category: string;
+}
+
+const StyledConstraint = styled(Box, {
+  shouldForwardProp: (prop) => prop !== "category",
+})<StyledConstraintProps>(({ theme, category }) => ({
+  borderLeft: `5px solid ${CATEGORY_COLORS[category]}`,
+  paddingRight: 0,
+  width: "100%",
+  color: theme.palette.text.primary,
+  position: "relative",
+  "&::after": {
+    content: "''",
+    position: "absolute",
+    bottom: 0,
+    left: 0,
+    width: "100%",
+    height: "1px",
+    background: theme.palette.secondary.main,
+  },
+}));
+
+interface ConstraintsListProps {
+  data: Constraint[];
+  metadata: GISResponse["metadata"];
+}
+
+export default function ConstraintsList({
+  data,
+  metadata,
+}: ConstraintsListProps) {
+  const groupedConstraints = groupBy(data, (constraint: Constraint) => {
+    return constraint.category;
+  });
+
+  return (
+    <Box mb={3}>
+      <List dense disablePadding>
+        {Object.keys(groupedConstraints).map((category: string) => (
+          <ListSubheader
```

Comment:
This element is causing a11y issues when checked agains the axe DevTools chrome extension - 

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/b8995c8e-d2b1-44ee-b3b8-a0d1b05aa7f0)

I think it's a simple fix though - `ListSubheader` should just wrap the `Typography`, and not all list `ConstraintListItem`, and I guess it should not have `component="div"`. That feels pretty weird as it's not really a list item but something separate - maybe there's some sort of aria property we can assign to make this clearer.

Reply:
Thanks for spotting this one! Re-arranged as suggested and errors are cleared up when scanning the component with closed/not-expanded items and when expanded now :heavy_check_mark: 

![Screenshot from 2023-06-12 10-14-51](https://github.com/theopensystemslab/planx-new/assets/5132349/c6ddb98d-727c-44fd-8b63-507ee7dd051f)